src/components/previewers/ImagePreviewer.jsx

```
@@ -0,0 +1,150 @@
+import React, { useCallback, useMemo, useState } from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)